### PR TITLE
update project to Swift 3

### PR DESCRIPTION
**Pull request by chrisdoc**
This PR updates STZPopupView to use Swift 3

### CODE REVIEW

File: Example/ViewController.swift [hunk 1]
```diff
@@ -96,14 +97,14 @@ class ViewController: UIViewController {
...
 
         // Close button
-        let button = UIButton(type: .System)
-        button.frame = CGRectMake(60, 60, 80, 40)
-        button.setTitle("Close", forState: UIControlState.Normal)
-        button.addTarget(self, action: "touchClose", forControlEvents: UIControlEvents.TouchUpInside)
+        let button = UIButton(type: .system)
+        button.frame = CGRect(x: 60, y: 60, width: 80, height: 40)
+        button.setTitle("Close", for: UIControlState())
+        button.addTarget(self, action: #selector(ViewController.touchClose), for: UIControlEvents.touchUpInside)
```
Comment (STAR-ZERO) [inline Example/ViewController.swift]: I prefer `#selector(touchClose)` to `#selector(ViewController.touchClose)`.

File: Source/UIViewController+STZPopupView.swift [hunk 2]
```diff
@@ -21,16 +21,16 @@ extension UIViewController {
     // MARK: - Property
     
     /// Popup target view
-    private var targetView: UIView {
+    fileprivate var targetView: UIView {
```
Comment (STAR-ZERO) [inline Source/UIViewController+STZPopupView.swift]: In this case, `private` is better than `fileprivate`.

File: Source/UIViewController+STZPopupView.swift [hunk 3]
```diff
@@ -82,10 +82,10 @@ extension UIViewController {
         let dismissButton = UIButton(frame: targetView.bounds)
         containerView.addSubview(dismissButton)
         if config.dismissTouchBackground {
-            dismissButton.addTarget(self, action: Selector("dismissPopupView"), forControlEvents: UIControlEvents.TouchUpInside)
+            dismissButton.addTarget(self, action: #selector(UIViewController.dismissPopupView), for: UIControlEvents.touchUpInside)
```
Comment (STAR-ZERO) [inline Source/UIViewController+STZPopupView.swift]: I prefer `#selector(dismissPopupView)` to `#selector(UIViewController.dismissPopupView)`.

File: Source/UIViewController+STZPopupView.swift [hunk 4]
```diff
@@ -98,30 +98,30 @@ extension UIViewController {
...
+            case .custom:
                 if let containerView = containerView, let popupView = popupView {
                     config.showCustomAnimation(containerView, popupView, { self.completionShowAnimation(true) })
                 }
             }
         }
     }
 
-    private func completionShowAnimation(finished: Bool) {
+    fileprivate func completionShowAnimation(_ finished: Bool) {
```
Comment (STAR-ZERO) [inline Source/UIViewController+STZPopupView.swift]: I prefer not to use underscore.
`private func completionShowAnimation(finished: Bool)`

File: Source/UIViewController+STZPopupView.swift [hunk 5]
```diff
@@ -136,7 +136,7 @@ extension UIViewController {
         dismissAnimation()
     }
 
-    private func completionDismissAnimation(finished: Bool) {
+    fileprivate func completionDismissAnimation(_ finished: Bool) {
```
Comment (STAR-ZERO) [inline Source/UIViewController+STZPopupView.swift]: I prefer not to use underscore.
`private func completionDismissAnimation(finished: Bool)`

File: Source/UIViewController+STZPopupView.swift [hunk 6]
```diff
@@ -67,7 +67,7 @@ extension UIViewController {
     - parameter popupView: Popup view
     - parameter config:    Config (Option)
     */
-    public func presentPopupView(popupView: UIView, config: STZPopupViewConfig = STZPopupViewConfig()) {
+    public func presentPopupView(_ popupView: UIView, config: STZPopupViewConfig = STZPopupViewConfig()) {
```
Comment (STAR-ZERO) [inline Source/UIViewController+STZPopupView.swift:70]: I prefer not to use underscore. 
`public func present(popupView: UIView, config: STZPopupViewConfig = STZPopupViewConfig())`

See https://developer.apple.com/reference/uikit/uiviewcontroller/1621380-present

API interface will change. It cannot be helped...

**Review by STAR-ZERO — CHANGES_REQUESTED** [review]
Thank you for PR.

There are several request changes.

**Comment by chrisdoc** [comment]
@STAR-ZERO thanks for the review, i fixed your remarks

**Comment by STAR-ZERO** [comment]
I'm sorry, I didn't explain it clearly enough.
- Please change all `fileprivate` to `private` in `Source/UIViewController+STZPopupView.swift`

I'm sorry, come to think of it, your first PR code is correct.
- Please use underscore at https://github.com/STAR-ZERO/STZPopupView/pull/6/commits/74187f24863f176802f55e0280a55105665c23ef#diff-a95f0f383b64e6ebc7aa159259f33594R70
  - before: `presentPopupView(popupView: UIView, config: STZPopupViewConfig = STZPopupViewConfig())`
  - after: `presentPopupView(_ popupView: UIView, config: STZPopupViewConfig = STZPopupViewConfig())`

Thanks:bow:

**Comment by chrisdoc** [comment]
@STAR-ZERO  no problem, I amended my `_` changes and replaced `fileprivate` with `private`

**Review by STAR-ZERO — APPROVED** [review]

**Comment by STAR-ZERO** [comment]
Thanks :smile::thumbsup:

**Pull request merged**